sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
i used the table because i think it is more flexible for future use, static arrays will be messy if a lot of rewrites will be added. it's a simple table, no complex querying is added, so not a lot of overhead to sql